Updated index.md - mongoose primer (#21345)
mdn/content: files/en-us/learn/server-side/express_nodejs/mongoose/index.md

> _Mongoose_ requires a connection to a MongoDB database. You can `require()` and connect to a locally hosted database with `mongoose.connect()`, as shown below.
> 
> > **Note:** This is a trivial example and not a valid mongoDB connection. An explanation of how to set up a working MongoDB database will be covered in the next section.

Reviewer: We can't have this particular note because this IS a valid mongoDB connection to a locally hosted database, and using a local database is a reasonable option for testing, and even deployment. 

What I'll do is hint that we don't use this approach for the tutorial. Anyone who mistakes the primer for direct instructions will then get a hint that this is not the tutorial that we want them to follow.

```suggestion
_Mongoose_ requires a connection to a MongoDB database.
You can `require()` and connect to a locally hosted database with `mongoose.connect()` as shown below (for the tutorial we'll instead connect to an internet-hosted database).
```
